tests/ballerina-unit-test/src/test/resources/test-src/types/map/map-value.bal

```diff
@@ -13,7 +13,7 @@ function testMapWithMap() returns (string){
     return list2String;
 }
 
-function testMapWithAnyValue() returns (int){
+function testMapWithAnyValue() returns (int|error){
```

Comment:
Can we fix the formatting issue as well?

Reply:
Fixed in 42ca94a